lfs/scanner_git_test.go

```diff
@@ -0,0 +1,76 @@
+package lfs_test // to avoid import cycles
+
+// This is for doing complete git-level tests using test utils
+// Needs to be a separate file from scanner_test so that we can use a diff package
+// which avoids import cycles with testutils
+
+import (
+	"testing"
+
+	. "github.com/github/git-lfs/lfs"
```

Comment:
It's not necessary, it's just a pattern I've seen used in this case of avoiding import cycles with test utils. e.g. http://golang.org/src/strings/search_test.go